sale_order_type/views/account_move_views.xml

```diff
@@ -4,7 +4,7 @@
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
         <field name="arch" type="xml">
-            <field name="journal_id" position="before">
+            <field name="journal_id" position="after">
```

Comment:
Why changing this order?

Reply:
In the other way field is not displyed correctly